example/declarative.dart

```diff
@@ -14,14 +14,14 @@ init(GrinderContext context) {
 }
 
 @Task(
-    depends: const ['init'],
+    depends: const [init],
```

Comment:
Yep, I think we could show the resulting output of `grind -h` for the example pubspec, which will show how the resulting task names and descriptions.